src/Framework/TestSuite.php

```diff
@@ -107,12 +107,12 @@ class TestSuite implements IteratorAggregate, Reorderable, SelfDescribing, Test
     protected $foundClasses = [];
 
     /**
-     * @var null|array<ExecutionOrderDependency>
+     * @var null|list<ExecutionOrderDependency>
      */
     protected $providedTests;
 
     /**
-     * @var null|array<ExecutionOrderDependency>
+     * @var null|list<ExecutionOrderDependency>
```

Comment:
if you were to default this property to an empty array; would it be possible to remove the null and thus have a cleaner type?